app/controllers/v0/virtual_agent/virtual_agent_claim_controller.rb

```diff
@@ -7,30 +7,25 @@ module V0
   module VirtualAgent
     class VirtualAgentClaimController < ApplicationController
       include IgnoreNotFound
-
       rescue_from 'EVSS::ErrorMiddleware::EVSSError', with: :service_exception_handler
-
       before_action { authorize :evss, :access? }
-
       def index
+        kms = KmsEncrypted::Box.new(previous_versions: [{ key_id: Settings.lockbox.master_key }])
+        user_info = { first_name: current_user.first_name, last_name: current_user.last_name,
+                      ssn: kms.encrypt(current_user.ssn), icn: current_user.icn }
         claims, synchronized = service.all
-
         case synchronized
         when 'REQUESTED'
-          render json: {
-            data: nil,
-            meta: { sync_status: synchronized }
-          }
+          render json: { data: nil, meta: { sync_status: synchronized } }
         when 'FAILED'
           error = EVSS::ErrorMiddleware::EVSSError.new('Could not retrieve claims')
+          VirtualAgentStoreUserInfoJob.perform_async(user_info, 'claims', {})
           service_exception_handler(error)
```

Comment:
It looks like the kms argument is missing as a parameter to the `VirtualAgentStoreUserInfoJob` here. You could make it optional